src/repo_agent.h

```diff
@@ -83,7 +83,7 @@ class TritonRepoAgent {
     return model_fini_fn_;
   }
 
- private:
+ protected:
```

Comment:
Why switch to protected here? Do we inherit from `TritonRepoAgent` somewhere?

Reply:
Yes, I defined an internal repo agent [here](https://github.com/triton-inference-server/core/pull/64/files#diff-8e4a56a97afaf3f9f3d0226a03e6a0eee2a373acc2e95663628b077a152e0076R93)